--- origsrc/Etoile-0.4.1/Frameworks/AddressesKit/Frameworks/AddressView/GNUmakefile	2009-03-17 08:17:59.000000000 -0500
+++ src/Etoile-0.4.1/Frameworks/AddressesKit/Frameworks/AddressView/GNUmakefile	2009-08-07 03:22:45.809987100 -0500
@@ -32,6 +32,7 @@ AddressView_LANGUAGES=Dutch English Fren
 
 AddressView_OBJCFLAGS=-I..
 AddressView_LDFLAGS=-L..
+AddressView_TOOL_LIBS += -lAddresses
 
 -include GNUmakefile.preamble
 -include GNUmakefile.local
--- origsrc/Etoile-0.4.1/Frameworks/MediaKit/GNUmakefile	2009-03-17 08:18:07.000000000 -0500
+++ src/Etoile-0.4.1/Frameworks/MediaKit/GNUmakefile	2009-08-07 03:22:45.821237200 -0500
@@ -15,8 +15,8 @@ MediaKit_HEADER_FILES = \
 				MKSoundDevice.h
 
 ADDITIONAL_OBJCFLAGS = -std=c99 -g -Werror `pkg-config libavcodec --cflags`
-ADDITIONAL_OBJCFLAGS += `pkg-config libavcodec --cflags`
-ADDITIONAL_OBJCFLAGS += `pkg-config libavformat --cflags`
+ADDITIONAL_OBJCFLAGS += -I`pkg-config libavcodec --variable includedir`/libavcodec
+ADDITIONAL_OBJCFLAGS += -I`pkg-config libavformat --variable includedir`/libavformat
 
 #If you install OSS 4 in Ubuntu, the header files are located here:
 ADDITIONAL_OBJCFLAGS += -I/usr/lib/oss/include
--- origsrc/Etoile-0.4.1/Frameworks/PopplerKit/config.sh	2009-03-17 08:18:13.000000000 -0500
+++ src/Etoile-0.4.1/Frameworks/PopplerKit/config.sh	2009-08-07 03:22:45.828737300 -0500
@@ -22,7 +22,7 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 POPPLER_CFLAGS="${POPPLER_CFLAGS} `${PKG_CONFIG} --cflags fontconfig`"
-POPPLER_LIBS="${POPPLER_LDFLAGS} `${PKG_CONFIG} --libs fontconfig`"
+POPPLER_LIBS="${POPPLER_LIBS} `${PKG_CONFIG} --libs fontconfig`"
 
 # poppler splash device
 ${PKG_CONFIG} --exists poppler-splash
@@ -31,7 +31,7 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 POPPLER_CFLAGS="${POPPLER_CFLAGS} `${PKG_CONFIG} --cflags poppler-splash`"
-POPPLER_LIBS="${POPPLER_LDFLAGS} `${PKG_CONFIG} --libs poppler-splash`"
+POPPLER_LIBS="${POPPLER_LIBS} `${PKG_CONFIG} --libs poppler-splash`"
 
 # poppler cairo device
 ${PKG_CONFIG} --exists poppler-cairo
